.travis.yml

```diff
@@ -81,9 +81,6 @@ branches:
   except:
     - /wip/
 notifications:
-  irc:
```

Comment:
[optional] I think the gitter link can be removed as well. Nobody noticed the lack of notification there. Also majority of the CI is done in github action, so the failure or success of ARM job not that relevant either.

Reply:
I opened this one for testing purposes (we had to reregister Travis CI for the syslog-ng repo), but @OverOrion reviewed it, so I thought "let's do it then" :) I don't want to dismiss the 2 approvals on this PR, but I support the idea of removing the Gitter notification, if someone wants to do it.